### PR TITLE
Move prometheus metrics for docker operations into dockershim

### PR DESCRIPTION
**Pull request by yujuhong**

### CODE REVIEW

File: pkg/kubelet/dockershim/metrics/metrics.go [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+	registerMetrics.Do(func() {
+		prometheus.MustRegister(DockerOperationsLatency)
+		prometheus.MustRegister(DockerOperations)
+		prometheus.MustRegister(DockerOperationsErrors)
+		prometheus.MustRegister(DockerOperationsTimeout)
+	})
+}
+
+// SinceInMicroseconds gets the time since the specified start in microseconds.
+func SinceInMicroseconds(start time.Time) float64 {
```
Comment (mtaufen) [inline pkg/kubelet/dockershim/metrics/metrics.go]: Is this unused?

Comment (mtaufen) [reply pkg/kubelet/dockershim/metrics/metrics.go]: Rest lgtm

Comment (mtaufen) [reply pkg/kubelet/dockershim/metrics/metrics.go]: cool

**Comment by mtaufen** [comment]
/assign @vishh 
for approval

**Comment by yujuhong** [comment]
Rebased (just a bazel BUILD file update) and re-applied lgtm

**Pull request merged**